code-challenges.test.js

```diff
@@ -13,47 +16,151 @@
+  });
+});
 
 // b) Create the function that makes the test pass.
+// i want to make a function inside of a conditional statement i can use for any number. i think it is a called a dynamic code. i will name it divByThree and make the if statement to return if is divisible by three or else return the statement that it is not divisible by three.
+const divByThree = (number) => {
+  if (number % 3 === 0) {
+    return `${number} is divisible by three`;
+  } else if (number % 3 !== 0) {
+    return `${number} is not divisible by three`;
+  }
+};
+// something wrong with jest, or my code.. but test is not running properly. This is what the terminal is returning..
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+// Your test suite must contain at least one test.
+
+//   at onResult (node_modules/@jest/core/build/TestScheduler.js:133:18)
+//   at node_modules/@jest/core/build/TestScheduler.js:254:19
+//   at node_modules/emittery/index.js:363:13
+//       at Array.map (<anonymous>)
+//   at Emittery.emit (node_modules/emittery/index.js:361:23)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       0 total
+// Snapshots:   0 total
+// Time:        0.384 s
+// Ran all test suites.
+// error Command failed with exit code 1.
+// info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
+// it looks like it is stating that i dont have a test? not sure what that means or how to fix it. im thinking that jest must be testing the wrong file. will cd into specific file "code-challenges.test.js and try again"
+// recieved the same message. i hope i can get clearaty for why this isn't working this upcoming week. i must admit, i am not feeling very good about this assesment, because of this result.
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
-
+//input= array of words
+//Expected output= array with all the words capitalized
+// i plan on making a const that will take in an array of words and using a for loop return the array with the words in caps.
+// const capArray = array.map((element) => {
+//   return element.toUpperCase();
+// });
 // a) Create a test with expect statements for each of the variables provided.
-
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+//created the parameters of the test to include what are we testing and the expected output to be. as i stated earlier, I'm not sure why jest is not functioning properly. but i hope the rest of my answer is correct.
+describe("capArray", () => {
+  it("takes an array of words and returns an array with the same words capitalized", () => {
+    expect(capArray(randomNouns1)).toEqual(
+      "Streetlamp",
+      "Potato",
+      "Teeth",
+      "Conclusion",
+      "Nephew"
+    );
+    expect(capArray(randomNouns2)).toEqual(
+      "Temperature",
+      "Database",
+      "Chopsticks",
+      "Mango"
+    );
+  });
+});
```

Comment:
This test looks super close!  Consider the data type you would want the output to be.  Hint: the prompt says "returns an array with all the words capitalized."